ansible/roles/operator-pipeline/templates/openshift/tasks/upload-signature.yml

```diff
@@ -0,0 +1,73 @@
+---
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  name: upload-signature
+spec:
+  params:
+    - name: docker_reference
+    - name: manifest_digest
+    - name: repository
```

Comment:
Some descriptions would be nice to have here. Especially if they contain examples. Got anything?

Reply:
For sure. Updated.